homework1/src/RemoveDuplicates.java

```diff
@@ -0,0 +1,17 @@
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedHashSet;
+
+public class RemoveDuplicates {
+    public static void main(String[] args) {
+        Integer[] numbers = {7, 10, 11, 15, 2, 3, 3, 10, 11, 20};
+        Arrays.sort(numbers, Collections.reverseOrder());
```

Comment:
smart solution

Reply:
but we don't need to sort the array, only reverse order

Reply:
![image](https://user-images.githubusercontent.com/30260615/100679261-2a185400-3380-11eb-9845-71faf4b3b56f.png)

Reply:
reverse order doesn't mean we need to sort it

Reply:
but I will assume that's not an issue, I'll close it